doc/source/management/data_at_rest_encryption.rst

```diff
@@ -0,0 +1,213 @@
+.. _data_at_rest_encryption:
+
+=======================
+Data at Rest Encryption
+=======================
+
+This feature is considered **BETA** quality.
+
+.. contents::
+   :local:
+
+.. _innodb_general_tablespace_encryption:
+
+InnoDB general tablespace encryption
+====================================
+
+In |Percona Server| :rn:`5.7.20-18` existing tablespace encryption support
+has been extended to handle general tablespaces. A general tablespace is either
+fully encrypted, covering all the tables inside, either not encrypted at all.
+It is not possible to have only some of the tables in the general tablespace
+encrypted.
+
+This feature extends the  `CREATE TABLESPACE
+<https://dev.mysql.com/doc/refman/5.7/en/create-tablespace.html>`_
+statement to accept the ``ENCRYPTION='Y/N'`` option.
+
+Prerequisites
+-------------
+
+This feature requires for either `keyring_file plugin
```

Comment:
This feature requires a keyring plugin, for example keyring_file or keyring_vault, to be loaded before it can be used.